### PR TITLE
Update Support Policy

### PR DESCRIPTION
**Pull request by xorima**
## Description

This commit updates our support policy for both operating systems and cookbook support

Signed-off-by: Jason Field <jason@avon-lea.co.uk>

### Issues Resolved

[List any existing issues this PR resolves]

### Check List
- [ ] All tests pass. See https://github.com/sous-chefs/apache2/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

File: content/docs/cookbook-support.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Cookbook support is a complicated subject based on many variables, however this document should outline expectations in our cookbooks
+
+## Semvar
+
+We follow semvar, this is never optional to our cookbooks, and allows you, our consumers, to understand the change we are making and the potential impact on your environment.
+
+## Cookbook Support
+
+We will attempt to support the version of the `chef-client` that chef support unless there is a compelling reason not to, such as:
```
Comment (JohnRoesler) [inline content/docs/cookbook-support.md]:
```suggestion
We will attempt to support the version of the `chef-client` that chef supports unless there is a compelling reason not to, such as:
```

File: content/docs/cookbook-support.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Features we require that are only in a later version of the chef client
+- Bugs which reqiure a certian version of the chef client
+- Ruby versions which are closely tied to chef version
+
+However as always we wecome pull-requests to extend our support across multiple versions
```
Comment (JohnRoesler) [inline content/docs/cookbook-support.md]:
```suggestion
However, as always, we welcome pull-requests to extend our support across multiple versions
```

File: content/docs/best-practices.md [hunk 3]
```diff
@@ -7,20 +7,26 @@ If we're adopting a cookbook and trying to bring it up to scratch, these are lik
...
-- Debian 8/9
-- Ubuntu 16.04/18.04
-- Amazon 1 and 2
+Current major Operating System support lies in the following:
+
+- Centos
+- Ubuntu
+- Redhat
+- Amazon Linux 2
+- Macos
```
Comment (tas50) [inline content/docs/best-practices.md]:
```suggestion
- macOS
```

File: content/docs/cookbook-support.md (hunk 1 above)
Comment (tas50) [inline content/docs/cookbook-support.md]:
```suggestion
We will attempt to support the version of the Chef Infra Client that Chef Software supports unless there is a compelling reason not to, such as:
```

File: content/docs/cookbook-support.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+## Semvar
+
+We follow semvar, this is never optional to our cookbooks, and allows you, our consumers, to understand the change we are making and the potential impact on your environment.
+
+## Cookbook Support
+
+We will attempt to support the version of the `chef-client` that chef support unless there is a compelling reason not to, such as:
+
+- Features we require that are only in a later version of the chef client
+- Bugs which reqiure a certian version of the chef client
```
Comment (tas50) [inline content/docs/cookbook-support.md]:
```suggestion
- Bugs which require a certain version of the Chef Infra Client
```

File: content/docs/cookbook-support.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+## Semvar
+
+We follow semvar, this is never optional to our cookbooks, and allows you, our consumers, to understand the change we are making and the potential impact on your environment.
+
+## Cookbook Support
+
+We will attempt to support the version of the `chef-client` that chef support unless there is a compelling reason not to, such as:
+
+- Features we require that are only in a later version of the chef client
```
Comment (tas50) [inline content/docs/cookbook-support.md]:
```suggestion
- Features we require that are only in a later version of the Chef Infra Client
```

File: content/docs/cookbook-support.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+- Bugs which reqiure a certian version of the chef client
+- Ruby versions which are closely tied to chef version
```
Comment (tas50) [inline content/docs/cookbook-support.md]:
```suggestion
- Ruby versions which are closely tied to Chef Infra Client versions
```

**Review by JohnRoesler — DISMISSED** [review]

File: content/docs/best-practices.md [hunk 7]
```diff
@@ -7,20 +7,26 @@ If we're adopting a cookbook and trying to bring it up to scratch, these are lik
 
 ## Platform Support
 
-The majority of our survey userbase uses the following and as such as aim to support these platforms by default:
+We will attempt to support the version of operating systems currently in support except where there is a compelling reason not to.
```
Comment (tas50) [inline content/docs/best-practices.md]:
```suggestion
We will attempt to support all operating systems releases currently supported by the OS vendor except where there is a compelling reason not to.
```

File: content/docs/best-practices.md [hunk 8]
```diff
@@ -7,20 +7,26 @@ If we're adopting a cookbook and trying to bring it up to scratch, these are lik
...
 
-- Centos 6/7
-- Debian 8/9
-- Ubuntu 16.04/18.04
-- Amazon 1 and 2
+Current major Operating System support lies in the following:
```
Comment (tas50) [inline content/docs/best-practices.md]:
```suggestion
Current major operating system support lies in the following:
```

File: content/docs/best-practices.md [hunk 9]
```diff
@@ -7,20 +7,26 @@ If we're adopting a cookbook and trying to bring it up to scratch, these are lik
...
+
+- Centos
+- Ubuntu
+- Redhat
+- Amazon Linux 2
+- macOS
+- Windows (Latest)
```
Comment (tas50) [inline content/docs/best-practices.md]:
```suggestion
- Windows
```

File: content/docs/best-practices.md [hunk 10]
```diff
@@ -7,20 +7,26 @@ If we're adopting a cookbook and trying to bring it up to scratch, these are lik
...
+
+Some cookbooks are not designed to support all of these operating systems, to see a full list of supported operating systems please refer to the `kitchen.yml` in the cookbook in question
```
Comment (tas50) [inline content/docs/best-practices.md]:
```suggestion
Some cookbooks are not designed to support all of these operating systems. For a full list of supported operating systems please refer to the `kitchen.yml` in the cookbook in question
```

File: content/docs/best-practices.md [hunk 11]
```diff
@@ -51,3 +57,7 @@ README.md
   - resource title
   - a table of available properties
   - a set of examples using the resource
+
+## Semvar
+
+Our cookbooks must follow semvar <semvar.org> this allows users to understand what the increments mean and plan accordingly
```
Comment (tas50) [inline content/docs/best-practices.md]:
```suggestion
Our cookbooks follow semvar guidelines. See <semvar.org> for an overview. Semvar allows allows cookbook consumers to better understand what version increments mean and plan their testing regiment accordingly
```

File: content/docs/best-practices.md [hunk 12]
```diff
@@ -51,3 +57,7 @@ README.md
   - resource title
   - a table of available properties
   - a set of examples using the resource
+
+## Semvar
```
Comment (majormoses) [inline content/docs/best-practices.md]:
```suggestion
## Semver
```

File: content/docs/best-practices.md [hunk 13]
```diff
@@ -51,3 +57,7 @@ README.md
...
+
+Our cookbooks follow semvar guidelines. See <semvar.org> for an overview. Semvar allows allows cookbook consumers to better understand what version increments mean and plan their testing regiment accordingly
```
Comment (majormoses) [inline content/docs/best-practices.md]:
```suggestion
Our cookbooks follow semver guidelines. See [the semver spec](https://semver.org/spec/v2.0.0.html) for an overview. Semver allows allows cookbook consumers to better understand what version increments mean and plan their testing regiment accordingly
```

File: content/docs/cookbook-support.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
+---
+title: 'Cookbook Best Practices'
+date: 2020-04-03T00:00:00+00:00
+---
+
+Cookbook support is a complicated subject based on many variables, however this document should outline expectations in our cookbooks
+
+## Semvar
```
Comment (majormoses) [inline content/docs/cookbook-support.md]:
```suggestion
## Semver
```

File: content/docs/cookbook-support.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+
+We follow semvar, this is never optional to our cookbooks, and allows you, our consumers, to understand the change we are making and the potential impact on your environment.
```
Comment (majormoses) [inline content/docs/cookbook-support.md]:
```suggestion
We follow semver, this is never optional to our cookbooks, and allows you, our consumers, to understand the change we are making and the potential impact on your environment.
```

**Review by majormoses — DISMISSED** [review]
LGTM

It's "semver" not "semvar"

**Review by tas50 — APPROVED** [review]

**Review by JohnRoesler — APPROVED** [review]

**Pull request merged**